src/Data/Formatter/DateTime.purs

```diff
@@ -107,9 +113,18 @@ printFormatterCommand = case _ of
   Milliseconds -> "SSS"
   Placeholder s -> s
 
+-- | The format string representation of a `Formatter`.
+-- | 
+-- | `show (Hours24 : MinutesTwoDigits : Nil) = "(Hours24 : MinutesTwoDigits : Nil)"`
+-- | 
+-- | while `printFormatter (Hours24 : MinutesTwoDigits : Nil) = "HHmm"`.
+-- | 
+-- | The interpretation of the format string is inspired by [momentjs](https://momentjs.com/docs/#/displaying/format/).
```

Comment:
I'd suggest we just mention this somewhere once, rather than on every function that deals with a format string. Probably in the same place we explain the format! 

We could perhaps the format info we have in the README just now in a module header comment, as that way it'll be contextually available in the pursuit docs too.